bert_e/settings.py

```diff
@@ -100,6 +100,7 @@ class SettingsSchema(Schema):
     need_author_approval = fields.Bool(missing=True)
     required_leader_approvals = fields.Int(missing=0)
     required_peer_approvals = fields.Int(missing=2)
+    pr_options = fields.Dict(missing={})
```

Comment:
I would rename this config with something in regards to the `author` as this option is not only about the PR but mostly about the author of a PR.